pyproject.toml

```diff
@@ -5,11 +5,14 @@ description = "Easily trim 'messages' arrays for use with GPTs."
 authors = ["Killian Lucas <killian@drinkwater.ai>"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.8"
```

Comment:
I wasn't able to add a new dependency via `poetry add` because `tiktoken` requires Python `3.8`.